debian/changelog

```diff
@@ -1,5 +1,6 @@
 python-ev3dev2 (2.0.0~beta3) stable; urgency=medium
 
+  * wait_until_not_moving should consider "running holding" as "not moving"
```

Comment:
Given that most of these changes are by you but I'm probably going to be the one doing the package release (and hopefully I'll get some time to add some changes of my own), I think we need section headers with the name of the person who made the changes.

Reply:
So something like
```
python-ev3dev2 (2.0.0~beta3) stable; urgency=medium

  * MoveJoyStick should use SpeedPercent
  * renamed GyroSensor rate_and_angle to angle_and_rate
 -- Daniel Walton <dwalton76@gmail.com>

  * foo
  * bar
 -- Kaelin Laundry <wasabifan@outlook.com>  Sat, 27 Oct 2018 21:18:00 -0700
```

I don't know if that falls into the standard debian changelog format (probably not) or if it matters if we deviate from the format.

Personally I am fine if my name is not associated with my commits in the changelog.  The commits in the git history will still show as being by me.  If I ever needed to reference work here when applying for a job, etc I would point people at the commit history since that is more detailed than the change log.

Reply:
Have a look back in history:

https://github.com/ev3dev/ev3dev-lang-python/blob/55f57ada176905a1db073423f428bd83dfd6b6ca/debian/changelog

Reply:
And yes, format matters very much in this file.

Reply:
Ack on format being important.  Lets do this...once this group of pull requests goes in I'll come back and update the changelog to add a ``[Daniel Walton]`` section for my commits.  That way I don't have to update 9 PRs.

Reply:
Sounds good.